README.md

```diff
@@ -251,6 +252,7 @@ ddev examples
 ddev format
 ddev gen-test-runner
 ddev saucelabs
+ddev task-runner
```

Comment:
I think I could go either way on this update. I'll let the other reviewers weigh in since it could theoretically be used outside of CI (even though I agree that is the most common use case)